Tests/SentryProfilerTests/SentryProfilerSwiftTests.swift

```diff
@@ -439,6 +439,10 @@ private extension SentryProfilerSwiftTests {
             try assertMetricValue(measurements: measurements, key: key, numberOfReadings: expectedUsageReadings, expectedValue: expectedUsage, transaction: transaction)
         }
 
+        let test: UInt? = nil
+        let crash = test!
+        print(crash)
```

Comment:
`h`: I guess you want to remove that before merging?

Reply:
yep!

Reply:
done, reverted the `DEBUG:` commit. see https://github.com/getsentry/sentry-cocoa/pull/3009/commits/c41e6c2e0cc888a8af52bb1c6ce6d26b875ad49b